workshops/food-and-beverage/optimized-distribution/Pipfile

```diff
@@ -0,0 +1,11 @@
+[[source]]
```

Comment:
Is this file/change really needed for logo modification?

Reply:
you were too fast. I removed that in another commit.